### PR TITLE
OSDOCS#8865 Installing extensions on a cluster in OLM 1.0

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.15

Issue: [OSDOCS-8865](https://issues.redhat.com//browse/OSDOCS-8865)

Link to docs preview: [Installing an Operator](https://71009--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/olm_v1/olmv1-installing-an-operator-from-a-catalog#olmv1-installing-an-operator_olmv1-installing-operator)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/olmv1-installing-an-operator.adoc [hunk 1]
```diff
@@ -7,40 +7,117 @@
...
-  version: 3.8.12
+  packageName: openshift-pipelines-operator-rh
+  channel: <channel>
+  version: <version>
 ----
++
+where:
++
+<channel>:: Optional: Specifies the channel, such as `pipelines-1.11` or `latest` for the package you want to install or update.
+<version>:: Optional: Specifies the specific version or version range, such as `1.11.1`, `1.12.x`, or `<1.13.1`, of the package you want to install or update.
```
Comment (adellape) [inline modules/olmv1-installing-an-operator.adoc:106]:
```suggestion
<version>:: Optional: Specifies the specific version or version range, such as `1.11.1`, `1.12.x`, or `1.13.1`, of the package you want to install or update.
```

File: modules/olmv1-installing-an-operator.adoc [hunk 2]
```diff
@@ -7,40 +7,115 @@
...
-  version: 3.8.12
+  packageName: openshift-pipelines-operator-rh
+  channel: <channel>
+  version: <version>
 ----
++
+where:
++
+<channel>:: Optional: Specifies the channel, such as `pipelines-1.11` or `latest` for the package you want to install or update.
+<version>:: Optional: Specifies the specific version or version range, such as `1.11.1`, `1.12.x`, or `>=1.12.1`, of the package you want to install or update. For more information, see "About target versions in OLM 1.0" and "Version range support".
```
Comment (michaelryanpeter) [inline modules/olmv1-installing-an-operator.adoc:106]: Note to reviewers: I will add the module for "Version range support" and add the xrefs to the content in a follow up PR.

**Review by everettraven — APPROVED** [review]
LGTM

**Comment by michaelryanpeter** [comment]
@jianzhangbjz PTAL :)

**Comment by michaelryanpeter** [comment]
@jianzhangbjz Friendly reminder to PTAL when you get the chance. Thank you!

**Comment by kuiwang02** [comment]
@michaelryanpeter just reminder:
it is on main branch and use https://github.com/openshift/openshift-docs/pull/71009/files#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R93. actually it is changed to clusterextensions.olm.operatorframework.io on 4.16.

so, the doc example apply for 4.15, and not 4.16
I say you add description `Version(s): 4.15` in this PR and label 4.15.
so, 
/lgtm

**Comment by michaelryanpeter** [comment]
Thank you, @kuiwang02! I saw the change to the `ClusterExtension` API in 4.16. Also, I will clean up the Quay Operator examples when I submit PRs for updating and deleting Operators so the examples in the doc are consistent throughout.

File: modules/olmv1-installing-an-operator.adoc [hunk 3]
```diff
@@ -5,42 +5,117 @@
...
+[source,text]
+----
+"latest"
+"pipelines-1.11"
+"pipelines-1.12"
+"pipelines-1.13"
+----
+====
+
+.. Get a list of versions published in a channel by running the following command:
```
Comment (mjpytlak) [inline modules/olmv1-installing-an-operator.adoc:58]:
```suggestion
.. Get a list of the versions published in a channel by running the following command:
```

File: modules/olmv1-installing-an-operator.adoc [hunk 4]
```diff
@@ -49,54 +124,113 @@ operator.operators.operatorframework.io/quay-example created
 +
 [source,terminal]
 ----
-$ oc get operator.operators.operatorframework.io/quay-example -o yaml
+$ oc get operator.operators.operatorframework.io pipelines-operator -o yaml
 ----
 +
+[NOTE]
+====
+If you specify a channel or define a version range in your Operator or extension's CR, {olmv1} does not display the resolved version installed on the cluster. Only the version and channel information specified in the CR are displayed.
```
Comment (mjpytlak) [inline modules/olmv1-installing-an-operator.adoc:132]: Same comment as line 14 about `{olmv1}`. It is already defined in this module, so not sure if there is another common attribute available to you when referencing subsequent instances.

File: modules/olmv1-installing-an-operator.adoc [hunk 5]
```diff
@@ -5,42 +5,117 @@
...
++
+<channel>:: Optional: Specifies the channel, such as `pipelines-1.11` or `latest` for the package you want to install or update.
+<version>:: Optional: Specifies the specific version or version range, such as `1.11.1`, `1.12.x`, or `>=1.12.1`, of the package you want to install or update. For more information, see "About target versions in OLM 1.0" and "Support for version ranges".
 
 . Apply the Operator CR to the cluster by running the following command:
 +
 [source,terminal]
 ----
-$ oc apply -f test-operator.yaml
+$ oc apply -f pipeline-operator.yaml
```
Comment (mjpytlak) [inline modules/olmv1-installing-an-operator.adoc:112]: Purely for consistency. Do you want to collapse the following example output? I see you doing that for net new content.

**Review by mjpytlak — COMMENTED** [review]
Hey @michaelryanpeter mostly all suggestions or observations, with exception to wondering if you need to keep you `{olmv1-first}` everywhere in the same module. Outside of the latter, feel free to take or leave what I commented on. LGTM.

File: modules/olmv1-installing-an-operator.adoc [hunk 6]
```diff
@@ -5,42 +5,117 @@
...
-= Installing an Operator
+= Installing an Operator from a catalog
 
-You can install an Operator from a catalog by creating an Operator custom resource (CR) and applying it to the cluster.
+{olmv1-first} supports installing Operators and extensions scoped to the cluster. You can install an Operator from a catalog by creating an Operator custom resource (CR) and applying it to the cluster.
+
+[IMPORTANT]
+====
+Currently, {olmv1-first} supports extensions that do not use webhooks and are configured to use the `AllNamespaces` install mode. Extensions that use webhooks or that target a single or specified set of namespaces cannot be installed.
+====
```
[the remark this answers is not in the document]
Comment (michaelryanpeter) [reply modules/olmv1-installing-an-operator.adoc:15]: Unfortunately, OLM v1 can only install extensions/Operators that support AllNamespace install mode AND don't use webhooks. At this point in the tech preview development cycle, Operators with webhooks or that are scoped to a specific namespace can't be installed.

Do you think that is clear enough in this admonition, @mjpytlak?

**Pull request merged**